connector-packager/tests/test_resources/oauth_connector/connectionResolver.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version='1.0' encoding='utf-8' ?>
+
+<tdr class='test_oauth'>
+  <connection-resolver>
+    <connection-builder>
+      <script file='connectionBuilder.js'/>
+    </connection-builder>
+    <connection-normalizer>
+      <required-attributes>
+        <attribute-list>
+            <attr>server</attr>
+            <attr>authentication</attr>
+            <attr>username</attr>
+            <attr>password</attr>
+            <attr>authentication-type</attr>
+            <attr>instanceurl</attr>
+            <attr>ACCESSTOKEN</attr>
+            <attr>REFRESHTOKEN</attr>
+            <attr>access-token-issue-time</attr>
+            <attr>access-token-expires-in</attr>
+            <attr>oauth-client</attr>
```

Comment:
```suggestion
            <attr>server</attr>
            <attr>port</attr>
            <attr>authentication</attr>
            <attr>username</attr>
            <attr>password</attr>
```
I believe the OAuth requires attributes are now handled internally.  Above should be correct.